From eadd168a5af7763e141a3a56da876ea820da920d Mon Sep 17 00:00:00 2001
From: thrishalig <thrishalig@ami.com>
Date: Fri, 18 Aug 2023 11:20:41 +0530
Subject: [PATCH] Changing unsigned to signed 8-bit sensor_min variable as it
 should take signed minimum value because temp sensors reading min range is
 having signed value(-128)

Signed-off-by: thrishalig <thrishalig@ami.com>
---
 sensorhandler.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sensorhandler.hpp b/sensorhandler.hpp
index 75212d8..6fc593d 100644
--- a/sensorhandler.hpp
+++ b/sensorhandler.hpp
@@ -328,7 +328,7 @@ struct SensorDataFullRecordBody
     uint8_t normal_max;
     uint8_t normal_min;
     uint8_t sensor_max;
-    uint8_t sensor_min;
+    int8_t sensor_min;
     uint8_t upper_nonrecoverable_threshold;
     uint8_t upper_critical_threshold;
     uint8_t upper_noncritical_threshold;
-- 
2.25.1

